### DISCUSSION TITLE
Add Direct Link to Current Live Page When in Edit Mode

### DISCUSSION THREAD

**Discussion by PatrickMessierGoC** [craftcms/cms#8913] [Ideas]
### Description
When a page is currently edited (modifed or in draft), there is no direct way to open the corresponding live page from the Edit Entry page. Editors some times need to toogle quickly between their current draft (using the Preview or Share button) and the corresponding live page. A link to the current live page (like the little planet icon on the Page Entry list) directly on the Edit Entry page would be handy.

Of course, editors can always open a second browser tab and navigate to the corresponding page but it seams that every click we can save them matters... 

Cheers

**Comment by brandonkelly** [comment]
Are you/they aware that the Share button will take you to the live page, if you are editing the “Current” revision, and the entry is actually live?

**Comment by PatrickMessierGoC** [comment]
Thanks for the quick reply. Yes, we are aware but that is not the workflow of our Editors and Publishers.

1.      Editors (users without publishing rights) save their work as draft. As soon as they do so, the Share button will link to the new Draft instead of the Live page.

2.      Once the Editors are happy with their draft, they submit them to Publishers (users with publishing rights). Publishers review the drafts and eventually publish them.

Since the Editors and Publishers are working with Drafts all along, the Share button cannot be used to quickly access the corresponding Live pages.

**Comment by mmikkel** [comment]
As for the Share button, in addition to not always linking to the live entry, the link is often tucked inside a dropdown (e.g. if you have multiple preview targets, or a popular plugin like SEOmatic installed that adds its own preview targets). It's both easy to learn and easy to forget that there's a link in there that (sometimes) goes to the current live entry 🤷‍♂️ 

When you create a new section, Craft automatically adds a "Link" column to the element index (there's also the similar "URI" column available). I don't see why entry edit pages shouldn't include a similar link somewhere, perhaps in the detail sidebar? A pretty basic convenience in my opinion (and I've absolutely seen content authors looking for that link when editing an entry).

**Comment by brandonkelly** [comment]
If the entry isn’t live, then there’s no way to link to the actual live entry. So we could either only show an additional link when the entry is actually live, or always show a link but it will act like the main share link on the Current revision – sometimes taking you to the live entry; sometimes taking you to a preview URL. Thoughts?

**Comment by PatrickMessierGoC** [comment]
For consistency sake with the entry list page, I would simply add a Link icon (little planet) somewhere that always links to the live page. If the page isn't live then a 404 would be returned. To avoid confusion, the Share button should be limited to share draft or unpublished pages only. That's how I see it.
